### ISSUE TITLE
action migration stalls on non-ASCII character in comment

### ISSUE THREAD

**Issue by nriley** [talonvoice/talon#347]
```
>>> actions.migrate.v02_all()
2021-07-10 17:33:38  INFO [v0.2] Starting batch migration: 
2021-07-10 17:33:40  INFO Created backup: /Users/nicholas/.talon/backup/user_2021-07-10_pTKD9i56.tar.gz
[...]
2021-07-10 17:33:40  INFO [v0.2] ++ user/knausj_talon/apps/platforms/mac/sublime_text/sublime_text_actions.py
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "talon/scripting/actions.py", line 75, in __call__
  File "talon/scripting/types.py", line 366, in call
  File "talon/scripting/core/migrate.py", line 237, in v02_all
  File "talon/scripting/actions.py", line 75, in __call__
  File "talon/scripting/types.py", line 366, in call
  File "talon/scripting/core/migrate.py", line 68, in v02_one
  File "lib/python3.9/encodings/ascii.py", line 26, in decode
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 244: ordinal not in range(128)
```

Seems to be coming from [this file](https://github.com/nriley/knausj_talon/blob/nriley/apps/mac/window_management.talon#L11) and the "é" in Exposé.

**Comment by lunixbochs** [comment]
What OS are you running the migration on? Python is supposed to be in UTF8 mode :(

**Comment by nriley** [comment]
This is macOS.

**Comment by nriley** [comment]
Fixed in 0.1.5-489, thanks.
